provisioning_templates/iPXE/ipxe_intermediate_script.erb

```diff
@@ -11,7 +11,7 @@ snippet: false
 :net<%= i %>
 isset ${net<%= i -%>/mac} || goto no_nic
 dhcp net<%= i -%> || goto net<%= i+1 %>
-chain <%= foreman_url('iPXE') %>?mac=${net<%= i -%>/mac} || goto net<%= i+1 %>
+chain <%= foreman_url('iPXE', mac: '${net0/mac}') %> || goto net<%= i+1 %>
```

Comment:
I believe this should have been:
```erb
chain <%= foreman_url('iPXE', mac: "${net#{i}/mac}") %> || goto net<%= i+1 %>

Reply:
Damn, thanks for that. https://github.com/theforeman/community-templates/pull/733